unit_tests/conftest.py

```diff
@@ -113,3 +113,22 @@ def fake_provisioner():  # pylint: disable=no-self-use
 @pytest.fixture(scope='session', autouse=True)
 def fake_region_definition_builder():  # pylint: disable=no-self-use
     region_definition_builder.register_builder(backend="fake", builder_class=FakeDefinitionBuilder)
+
+
+@pytest.fixture(scope="function", name="params")
```

Comment:
notice there's already a fixture like this here

pylint even warns about it:
```
unit_tests/conftest.py:119:0: E0102: function already defined line 54 (function-redefined)
```

I think the older once can be removed

Reply:
strange that when I commit it, the pylint did not warn

Reply:
removed